### PR TITLE
Changed all loading indicators

### PR DESCRIPTION
**Pull request by u22520016**
I changed all loading indicators to the triangle.

### CODE REVIEW

**Review by pieryvdl — APPROVED** [review]
Thanks Franco. I  think it will actually better the user's mood seeing the slower animation. Good work.

**Review by LeratoKgomo — APPROVED** [review]
looks great, thanks

**Comment by u22520016** [comment]
I had to fix merge conflicts that were really weird and interesting...please just check again @LeratoKgomo @pieryvdl

**Review by pieryvdl — COMMENTED** [review]
Hey Franco! I checked out the branch again. There is one loading indicator that I think was missed. On the View events page, the event image at the top still loads using circular progress indicator. Will see if I can comment it for you.

File: lib/view/components/events/joined_event_card.dart [hunk 1]
```diff
@@ -108,9 +115,10 @@ class _UpcomingEventCardWidgetState extends State<UpcomingEventCardWidget> {
                               ),
                               Text(
                                 'Upcoming: ${widget.e?.name}',
-                                style:  TextStyle(
+                                style: TextStyle(
                                   fontFamily: 'Inter',
-                                  color: Theme.of(context).brightness == Brightness.light
```
Comment (pieryvdl) [inline lib/view/components/events/joined_event_card.dart:113]: Above this comment on line 111 there is still a circular progress indicator.

**Review by LeratoKgomo — APPROVED** [review]

**Pull request merged**